src/Sulu/Bundle/SearchBundle/Tests/Functional/Controller/SearchControllerTest.php

```diff
@@ -43,9 +43,8 @@ class SearchControllerTest extends SuluTestCase
 
     public function setUp(): void
     {
-        parent::setUp();
```

Comment:
Can we just remove that?

Reply:
yes they is no parent implementation I removed it to be consistent